### ISSUE TITLE
Stable baselines not working for Reacher-v2 mujoco env

### ISSUE THREAD

**Issue by chaiban** [hill-a/stable-baselines#425]
This is the code I used for training Reacher environment. DDPG and PPO2

import gym
import numpy as np
from stable_baselines.ddpg.policies import MlpPolicy
from stable_baselines.common.vec_env import DummyVecEnv
from stable_baselines import DDPG

env = gym.make('Reacher-v2')
env = DummyVecEnv([lambda: env])

model = DDPG(MlpPolicy, env, verbose=1)
model.learn(total_timesteps=1e7)
model.save("ddpg_reach")

del model # remove to demonstrate saving and loading

model = DDPG.load("ddpg_reach")

obs = env.reset()
while True:
    action, _states = model.predict(obs)
    obs, rewards, dones, info = env.step(action)
    env.render()

I have trained for very long time, but algorithms doesn't seem to work for this environment.
What could be the reason ?

Thank you for your suggestions!

Result after training for 1e7 iterations

--------------------------------------
| reference_Q_mean        | -4.33    |
| reference_Q_std         | 2.98     |
| reference_action_mean   | 0.0381   |
| reference_action_std    | 0.149    |
| reference_actor_Q_mean  | -4.2     |
| reference_actor_Q_std   | 2.95     |
| rollout/Q_mean          | -2.69    |
| rollout/actions_mean    | 0.0154   |
| rollout/actions_std     | 0.149    |
| rollout/episode_steps   | 50       |
| rollout/episodes        | 2e+05    |
| rollout/return          | -10.2    |
| rollout/return_history  | -10.9    |
| total/duration          | 3.61e+04 |
| total/episodes          | 2e+05    |
| total/epochs            | 1        |
| total/steps             | 9999998  |
| total/steps_per_second  | 277      |
| train/loss_actor        | 4.19     |
| train/loss_critic       | 0.135    |
| train/param_noise_di... | 0        |
--------------------------------------

**Comment by araffin** [comment]
Hello,
You did not change the default hyperparameters and forgot to add noise for ddpg, so i would say it is an expected behavior.
Please use the rl zoo and look at the hyperparameters provided for pybullet envs, this should work for mujoco envs too.

PS: please use code block for formatting your code (cf issue template)

**Comment by chaiban** [comment]
Thank you for the details.
I did train it both ways with and without noise. It resulted in same output. 
I will look at rl zoo and hyperparameters. Thank you

**Comment by chaiban** [comment]
Hello @araffin ,
I trained with rl zoo as you pointed out. I doesn't seem to converge. 

As you mentioned, I tested with both mujoco and bullet reacher env. Bullet reacher env works well but mujoco env doesn't seem to work.

The parameters used with ppo2:
Reacher-v2:
  normalize: true
  n_envs: 8
  n_timesteps: !!float 2e6
  policy: 'MlpPolicy'
  n_steps: 2048
  nminibatches: 32
  lam: 0.95
  gamma: 0.99
  noptepochs: 10
  ent_coef: 0.0
  learning_rate: 2.5e-4
  cliprange: 0.2

Any other ideas on how to make it work ? 

Thank you

**Comment by araffin** [comment]
Hello,

Using those hyperparameters (copied from HalfCheetah), it seems to work quite quickly (average training reward of -5 in less than 2e5 steps):

```
Reacher-v2:
  env_wrapper: utils.wrappers.TimeFeatureWrapper
  normalize: true
  n_envs: 1
  n_timesteps: !!float 1e6
  policy: 'MlpPolicy'
  n_steps: 2048
  nminibatches: 32
  lam: 0.95
  gamma: 0.99
  noptepochs: 10
  ent_coef: 0.0
  learning_rate: !!float 3e-4
  cliprange: 0.2
```
